src/cli/README.md

```diff
@@ -807,6 +808,27 @@ Delete all stored settings, and signal a platform reset.
 > factoryreset
 ```
 
+### femparam
+
+Get the external FEM parameters.
+
+```bash
+> femparam
+LNA gain 11 dBm
+Done
```

Comment:
```suggestion
> fem lnagain
11
Done
```

Reply:
I know there are examples in CLI that input is a number without unit, and the output is a number with unit. 
I prefer input and output to be the same for consistency. Thoughts? @jwhui @abtink @brianljt

Reply:
This is inline with ccathreshold and txpower. I think the unit related to power is dBm for most of the cases so there are no need to add unit to the input number. Also, the unit is clarified in the README.md file for cli.